.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: node_js
+
 node_js:
-  - "0.10"
+  - "lts/*"
+  - "node"
```

Comment:
Doesn't have to be in this PR, but just dropping a note - I'm down to switch over to GitHub Actions. Haven't seen a `.travis.yml` update that wasn't a removal in quite a while 😄